packages/statemanager/src/stateManager.ts

```diff
@@ -667,7 +669,7 @@ export class DefaultStateManager implements StateManagerInterface {
     const trie = this._getStorageTrie(address)
     trie.root(hexToBytes(storageHash))
     for (let i = 0; i < storageProof.length; i++) {
-      await updateFromProof(
+      await updateTrieFromMerkleProof(
```

Comment:
Ah, especially this one I can understand a lot better just from reading it, now seeing it in the code base! 👍